packages/e2e-tests/test/util-helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function sleep(ms: number) {
```

Comment:
I could also inline it in the test's scope it but I figure there's a good chance this kind of file will be needed? At least I was looking for that.

Reply:
I wouldn't overthink it tbh – currently, we use inline code like `await new Promise((resolve) => setTimeout(resolve, 400));` in a bunch of tests, but this obviously also works.

We do tend to prefer adding TS return types wherever possible, by the way 🙂 Only reason it's not a linter rule is that the codebase currently still has a bunch of exceptions to that rule

Reply:
Sounds good, I just copy-pasted this from an old `node-runtime-worker-thread` spec and didn't change it. I like the readability of `sleep` though don't mind a promise either; maybe it's worth keeping? We use sleep in the worker thread tests anyhow.

Re: TS return types, great to hear! I love strong typing, just didn't think much about it here (and similarly influenced by the lack of it right now).

Reply:
Yeah, I can make a small chore PR later for that.